src/poetry/vcs/git/backend.py

```diff
@@ -331,16 +332,28 @@ def _clone_submodules(cls, repo: Repo) -> None:
         repo_root = Path(repo.path)
         modules_config = repo_root.joinpath(".gitmodules")
 
+        # A relative URL by definition starts with ../ or ./
+        relative_submodule_regex = re.compile("^(\\.\\./|\\./),?")
```

Comment:
whats the `,?` for?

The tests pass without it, so if it's doing something useful then you're missing a test.

A more direct translation of your words into regex would be `re.compile(r"^\.{1,2}/")`

Reply:
Yup, good catch - thanks. Resolved in latest commit.